boards/arm/b_u585i_iot02a/b_u585i_iot02a-common.dtsi

```diff
@@ -131,11 +131,10 @@
 
 	status = "okay";
 
-	mx25lm51245: ospi-nor-flash@0 {
+	mx25lm51245: ospi-nor-flash@70000000 {
 		compatible = "st,stm32-ospi-nor";
-		reg = <0>;
+		reg = <0x70000000 DT_SIZE_M(64)>; /* 512 Mbits */
 		ospi-max-frequency = <DT_FREQ_M(50)>;
-		size = <DT_SIZE_M(512)>; /* 64 MBytes */
```

Comment:
If this is also valid in non memmap cases, let's make it a dedicated PR, extended to all similar descriptions in STM32 boards

Reply:
During the MemoryMapped mode, the driver controls the external flash base address and size.
Both values are given by this <reg> property (better solution than using a definition in the board Kconfig).
The <reg> property exists. Here it is filled with actual values of the external flash when the external flash is accessed in memorymapped mode.
When the external flash is accessed in indirect mode (non-MemoryMapped), the base address is useless and the size is given by the <size> property.